x-pack/plugins/observability/public/pages/alerts/alerts_search_bar.tsx

```diff
@@ -25,6 +26,7 @@ export function AlertsSearchBar({
     dateRange: { from: string; to: string; mode?: 'absolute' | 'relative' };
     query?: string;
   }) => void;
+  addToQuery: (value: string) => void;
```

Comment:
Is this used anywhere?

Reply:
Nope, it is a leftover from something I was trying. I'll remove it